edx_proctoring/templates/proctored_exam/submitted.html

```diff
@@ -7,6 +7,11 @@
   </h3>
   <p>
     {% blocktrans %}
+    If your proctoring client is still open, please close it now.
```

Comment:
@dianakhuang Here is my suggestion, to more closely match what's in the current learner doc. Let me know what you think:
"If the proctoring software window is still open, you can close it now. Confirm that you want to quit the application when you are prompted."